projects/schematics/src/migrations/4_0/constructor-deprecations/data/configurator-cart.service.migration.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  ACTIVE_CART_SERVICE,
+  CHECKOUT_FACADE,
+  CHECKOUT_SERVICE,
+  NGRX_STORE,
+  PRODUCT_PAGE_EVENT_BUILDER,
+  SPARTACUS_CORE,
+  SPARTACUS_PRODUCT_CONFIGURATOR,
+  STORE,
+  USER_ID_SERVICE,
+} from '../../../../shared/constants';
+import { ConstructorDeprecation } from '../../../../shared/utils/file-utils';
+
+export const CONFIGURATOR_CART_SERVICE_MIGRATION: ConstructorDeprecation = {
+  // feature-libs/product-configurator/rulebased/core/facade/configurator-cart.service.ts
+  class: PRODUCT_PAGE_EVENT_BUILDER,
+  importPath: SPARTACUS_PRODUCT_CONFIGURATOR,
+  deprecatedParams: [
+    { className: STORE, importPath: NGRX_STORE },
+    { className: STORE, importPath: NGRX_STORE },
```

Comment:
Please remove duplicated line

Reply:
There are 2 `Store` types parameters in the `ConfiguratorCartService` constructor.  I thought they each need an entry in the params list to match that.  I'll try to confirm.

Reply:
I got the confirmation that one entry peer constructor parameter is required.